src/Native/Runtime/windows/PalRedhawkInline.h

```diff
@@ -40,6 +40,13 @@ FORCEINLINE Int32 PalInterlockedExchange(_Inout_ _Interlocked_operand_ Int32 vol
     return _InterlockedExchange((long volatile *)pDst, iValue);
 }
 
+EXTERN_C long __PN__MACHINECALL_CDECL_OR_DEFAULT _InterlockedExchangeAdd(long volatile *, long);
```

Comment:
I guess it came back when I was rebasing the previous commit to the currrent master. I remember removing it before.